src/status_im/contexts/communities/events.cljs

```diff
@@ -348,11 +348,7 @@
       (navigate-to-serialized-community community-id)
       (rf/merge
        cofx
-       {:fx [[:dispatch
-              [:communities/fetch-community
-               {:community-id           community-id
```

Comment:
hi @flexsurfer, Thank you for looking into this.

Just to be sure, did you tried opening universal links for unfetched communities?

Reply:
and community chats using universal links for unfetched communities?

Reply:
Same concern here, this is used by universal link to jump to community overview
unfetched community universal link will lead to infinite loading screen

I recommend checking if community already fetched using the community-id

Reply:
thanks

Reply:
@yqrashawn @Parveshdhull thank. made changes, pls take a look